dev-tools/packer/docker/xgo-image-deb6/beats-builder/Dockerfile

```diff
@@ -20,11 +20,18 @@ RUN ./fetch.sh http://www.tcpdump.org/release/libpcap-1.8.1.tar.gz 32d7526dde8f8
   ./configure --enable-usb=no --enable-bluetooth=no --enable-dbus=no && \
   make
 
-# Old git version which does not support proxy with go get requires to fetch go-yaml directly
-RUN git clone https://github.com/go-yaml/yaml.git /go/src/gopkg.in/yaml.v2
+# Old git version which does not support proxy with go get directly, and git cloning
+# from github also no longer works.
+ADD go-yaml.tar.gz /go-yaml.tar.gz
```

Comment:
I think we would have move visibility into this dependency if it were copied from the dev-tools/vendor directory. This will probably be better for maintainability.